src/Microsoft.ServiceFabric.Actors/Diagnostics/ActorMethodInfoUtil.cs

```diff
@@ -0,0 +1,40 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ServiceFabric.Actors.Runtime;
+using Microsoft.ServiceFabric.Services.Remoting.Description;
+
+namespace Microsoft.ServiceFabric.Actors.Diagnostics
+{
+    internal class ActorMethodInfoUtil
```

Comment:
Could you make this class static and remove the default internal visibility? We only need it for functions.